content/applications/marketing/events/registration_desk.rst

```diff
@@ -0,0 +1,109 @@
+=================
+Registration desk
+=================
+
+Use the *Registration Desk* feature in Odoo *Events* to grant event access to registered attendees
+as they arrive, and store valuable attendee-related data in the reporting metrics for future
+analysis.
+
+Registration Desk page
+======================
+
+First, the person (or people) in charge of granting access to event attendees, would ideally have a
+compatible mobile-device open to their Odoo database.
+
+.. note::
+   The *Registration Desk* feature works with laptops that have cameras, but mobile devices are more
+   convenient for managing attendee access.
+
+Then, open the :menuselection:`Events app`, and click the :guilabel:`Registration Desk` header menu
+option. Doing so, reveals a separate :guilabel:`Registration Desk` page.
+
+.. image:: registration_desk/registration-desk-page.png
+   :align: center
+   :alt: The Registration Desk page in the Odoo Events application.
+
+There is only a :guilabel:`Registration Desk` square in the center of the page, with a simple
+message, reading: :guilabel:`Welcome to (Company Name)`. Beneath that, the company logo that has
+been uploaded in the database appears.
```

Comment:
Not sure if necessary? Or can add this info as a more concise caption to the screenshot.